examples/20171108_Pasadena/run_example_modtran.sh

```diff
@@ -1,13 +0,0 @@
-# David R Thompson, Adam Erickson
```

Comment:
Moved to `isofit/test/test_examples.py`. I don't see a reason to maintain multiple versions of the configuration.